docs/docs/components/guidelines/README.md

```diff
@@ -249,7 +249,27 @@ constructed API requests to reduce code and improve maintenance.
 
 #### Pagination
 
-When making API requests, handle pagination to ensure all data / events are processed.
+When making API requests, handle pagination to ensure all data/events are
+processed. Moreover, if the underlying account experiences and/or generates too
+much data paginating through the entire collection of records might cause
```

Comment:
paginating through the entire collection of records might -> 

paginating through the entire collection of records, it might